src/Entities/Token.js

```diff
@@ -22,6 +26,22 @@ export default class Token {
     }
   }
 
+  resolveStorage(storageType) {
```

Comment:
wydaje mi się że ta metoda znajdować się gdzieś indziej jako że nie ma to za dużo wspólnego z tokenem samym w sobie, np. w klasie Storage czyli trzeba by przekazywać `storageType` do konstruktora od razu w 18 linijce - `this.storage = new Storage(storageType)`

Reply:
Poprawione, to jest jakaś moja ciężka próba implementacji wzorców ;)